docs/data/hubble/analytics-platforms.mdx

```diff
@@ -7,14 +7,15 @@ title: Ecosystem Data Analytics Platforms
+| Platform | Description | Home page and Signup | Pricing | Data Access | Dashboard access |
+| --- | --- | --- | --- | --- | --- |
+| SDF Provided Data Analytics (Native) | SDF provides a publicly available raw dataset with a complete historical record of Stellar network data, updated every 15 minutes. | N/A | End users incur the cost to execute queries. Visit the [BigQuery Pricing](https://cloud.google.com/bigquery/pricing#analysis_pricing_models) page to learn more. | Please see the [Analyst Guide](./analyst-guide) for more information. | N/A |
+| Dune | Raw dataset for the Stellar network available in the Dune portal. Updated every hour. You can compare different chains on the platform. | [Dune](https://dune.com/) <br></br>[Signup](https://dune.com/auth/register) | Free for simpler queries. Need to pay for complex queries. | To access datasets, click [here](https://dune.com/queries?category=canonical&namespace=stellar). | Explore community dashboards, network overview, and datasets [here](https://dune.com/blockchains/stellar). |
+| Artemis | Aggregated dataset for the Stellar network available in BigQuery as well as the Artemis portal. Provides dashboards to compare Stellar chain activity with ecosystem chains. | [Artemis](https://www.artemis.xyz/) <br></br>[Signup](https://app.artemisanalytics.com/artemis-terminal-auth?flow=SIGN_UP) | Free dashboards. Need to pay for accessing the raw dataset. | For accessing Artemis-transformed Stellar dataset, upgrade to the enterprise plan and then star [this dataset](https://console.cloud.google.com/bigquery?ws=!1m4!1m3!3m2!1sartemis-bigquery-share!2sshared_us). | Explore [chain compare](https://app.artemisanalytics.com/chains?selectedChains=stellar) and [application activity](https://app.artemis.xyz/application-activity/stellar) dashboards. |
+| Nansen | Aggregated dataset for the Stellar network available in the Nansen portal. Provides dashboards to compare Stellar chain activity with ecosystem chains. | [Nansen](https://app.nansen.ai/) <br></br>[Signup](https://app.nansen.ai/auth/signup) | Free dashboards. Need to pay for accessing the raw dataset. | For accessing Nansen-transformed Stellar dataset, upgrade to the Pioneer plan and then star [this dataset](https://console.cloud.google.com/bigquery?project=nansen-query). | Explore [chain activity, entity, and ecosystem](https://app.nansen.ai/macro/blockchains?chain=stellar&tab=overview). |
+| Ortege | N/A | [Ortege](https://www.ortege.ai/) <br></br>[Signup](https://app.ortege.ai/register/form) | N/A | N/A | N/A |
+| Flipside | N/A | [Flipside](https://flipsidecrypto.xyz/home/) <br></br>[Signup](https://flipsidecrypto.xyz/auth/sign-up) | Entire data access (raw, dashboards) is free. | To access the dataset, sign up on Flipside and find the "Stellar" database in [Flipside studio](https://flipsidecrypto.xyz/studio/). | Explore [community dashboards](https://flipsidecrypto.xyz/insights/dashboards?search=stellar). |
```

Comment:
Description: Raw dataset access to run queries, create dashboards and gain actionable intelligence on various crypto ecosystems. Flipside sponsors analytics bounties to encourage open source sharing of insights as well as developer quests.